### DISCUSSION TITLE
Password dialoge allowing users to paste from clipboard

### DISCUSSION THREAD

**Discussion by LimakPL** [ncruces/zenity#114] [Ideas]
Hi
is any way to allowing pasting password from clipboard to text field in password dialogue? For now is not possible, but it will much easer for users. I am using your app on macbooks.

**Comment by ncruces** [comment]
I'm just using the AppleScript password dialog. If that doesn't allow copy paste… there's nothing I can do about it.

**Comment by MinhNghiaD** [comment]
Hello, I'm having similar problem on Mac but it's not only for password. It seems like zenity text entry doesn't allow any Cmd combination. I tried Cmd+V, Cmd+C, Cmd+Z, Cmd+A, nothing works

**Comment by ncruces** [comment]
See: #121
